IowaPD_dispatch.yaml

```diff
@@ -469,6 +470,12 @@ entityDefinitions:
         column: Longitude
         transforms:
         - !<transforms.ParseDoubleTransform> {}
+      ol.locationcoordinates:
+        type: ol.locationcoordinates
+        transforms:
+        - !<transforms.ConcatTransform>
```

Comment:
We also have now:
```
- !<transforms.GeographyPointTransform>:
    latTransforms:
        - !<transforms.ColumnTransform>:
          column: Latitude
    lonTransforms:
        - !<transforms.ColumnTransform>:
          column: Longitude
```
It would be more lines here, but guarantees an error is thrown with invalid input...

Reply:
sounds god, I changed it. Can you check that the spacing is right in the code?